cmd/service/delete.go

```diff
@@ -43,7 +43,7 @@ func deleteHandler(cmd *cobra.Command, args []string) {
 			return
 		}
 		for _, service := range reply.Services {
-			args = append(args, service.Hash())
+			args = append(args, service.Id)
```

Comment:
ID comes from our custom Service type. Id comes from gRPC Service type because it cannot convert id field described in the proto file to full-uppercase.